docs/releases/5.1.txt

```diff
@@ -26,6 +26,20 @@ only officially support the latest release of each series.
 What's new in Django 5.1
 ========================
 
+Require authentication for all views by default
+-----------------------------------------------
+
+The new :class:`~django.contrib.auth.middleware.LoginRequiredMiddleware`
+redirects all unauthenticated requests to a login page. Views can allow
+unauthenticated requests by using the
+:func:`~django.contrib.auth.decorators.login_not_required` decorator.
+
+The :class:`~django.contrib.auth.middleware.LoginRequiredMiddleware` respects
+``login_url`` and ``redirect_field_name`` set via the
+:func:`~.django.contrib.auth.decorators.login_required` decorator, but does not
+support setting ``login_url`` or ``redirect_field_name`` via the
+:class:`~django.contrib.auth.mixins.LoginRequiredMixin`.
+
```

Comment:
Very optional, but would there be a way to grow this section with a small yet (ideally) complete example like we have done in the past for choices allowing a dict or db defaults?

Reply:
Not that I can think of?

Reply:
Fair enough, thank you! I really appreciate all the effort you've put into this.